build.sbt

```diff
@@ -12,7 +12,7 @@ licenses := Seq("Apache 2.0" -> url("https://raw.githubusercontent.com/amplab/ke
 
 homepage := Some(url("http://keystone-ml.org"))
 
-scalaVersion := "2.10.4"
+scalaVersion := "2.10.6"
```

Comment:
I don't think I have to have 2.11 support, but my version of spark is currently only set to build 2.11. It seems to work for me with mlmatrix 2.11 and setting keystoneml to scala 2.11, I just had to comment out NERSuite and POSTaggerSuite because the library(ies?) those depend on doesn't seem to work.